### ISSUE TITLE
Some Bipod positions are doing weird things...

### ISSUE THREAD

**Issue by Saborknight** [gruppe-adler/grad_trenches#111]
GRAD Trenches v1.6.8.6

So some of the bipod positions either don't offer the full arc of fire, or bipod you 1m to left/right of where you are (a significant distance anyway), and on the small trench there's this hump in the entrance making the player elevate as high as the trench is when crawling in from the back?

Video of what I mean.
https://youtu.be/c3aClDGHCdw

**Comment by nomisum** [comment]
this worked before the last update, right? i wonder if its more a model or more an animation system issue 🤔

**Comment by Saborknight** [comment]
Eh, I don't know about specific, but ever since the major(-ish?) Update where some of the trench types and models changed we've been having troubles. We noticed that you had updated the model collisions so that they have more space before the visible model which could be used for the bipod, but the results are as you see in the video there. Good attempt though.

Our solution has been to build the trenches 3/4 of the way up and then leave them like that. Some of our guys have had more success with the trenches like that.

**Comment by nomisum** [comment]
thanks, that is enough context for @AdlerSalbei to investigate i guess

**Comment by Saborknight** [comment]
I know it's not the most helpful but I was surprised no one mentioned it before

**Comment by AdlerSalbei** [comment]
Yeah, it's a know problem with the Geo LOD. It's a real pain in the ass to fix. Since I have to change the GEO LOD's height to fit to the Arma Algorithm
